docs/quick-start-demo.md

```diff
@@ -15,98 +15,133 @@ keywords:
-- Privado ID Wallet App is an implementation of the [<ins>Wallet SDK</ins>](/docs/category/wallet-sdk), as a way of showcasing its possibilities. Learn more about the Wallet App [<ins>here</ins>](/docs/wallet/wallet-app/privadoid-app.md).
+The Privado ID Web Wallet is a web-based identity wallet. It is a reference implementation built using our [<ins>JS SDK</ins>](/docs/js-sdk/js-sdk-overview.md). [<ins>Learn more about the Web Wallet</ins>](/docs/wallet/web-wallet.md).
 :::
 
-## Issue a new credential to attest to the ID Holder's event attendance
+</TabItem>
+
+<TabItem value="Mobile App">
+
+- Android: <a href="https://play.google.com/store/apps/details?id=id.privado.wallet" target="_blank">Privado ID Wallet App on Google Play</a>
+- iOS: <a href="https://apps.apple.com/us/app/privadoid/id6566184703" target="_blank">Privado ID Wallet App on the App Store</a>
+
+:::note
+Privado ID Wallet App is an implementation of the [<ins>Wallet SDK</ins>](/docs/category/wallet-sdk), as a way of showcasing its possibilities. [<ins>Learn more about the Wallet App</ins>](/docs/wallet/wallet-app/privadoid-app.md).
+:::
 
-A trusted entity, for instance, a private institution will now play the role of an issuer. It will be responsible for creating the credential and sending it to the ID Holder.
+</TabItem>
+</Tabs>
 
-We are using <a href="https://issuer-demo.privado.id/">the Issuer Node UI Demo environment</a> to manage credentials. This is the place where you as an issuer can can create and manage identities and credentials, generate connections and manage schemas.
 
-However, if you are using a new credential type, you actually need to create a schema for that credential, which basically is the set of JSON files that gather all the attributes of that specific credential.
+## Issue the POAP credential
 
-To facilitate this issuance process, we have already created the credential schema with schema type POAP01 whose URLs are the following:
+You will now take the role of an Issuer, in this example, the private institution/trusted entity that held the event and creates a POAP. The Issuer is responsible for both creating the credential and sending it to the ID Holder.
 
-- JSON schema URL
-  `ipfs://QmTSwnuCB9grYMB2z5EKXDagfChurK5MiMCS6efrRbsyVX`
-- JSON-LD Context
-  `ipfs://QmdH1Vu79p2NcZLFbHxzJnLuUHJiMZnBeT7SNpLaqK7k9X`
+We will use <a href="https://issuer-demo.privado.id/">the Issuer Node UI Demo environment</a> to manage credentials. This environment enables an Issuer to create and manage identities and credentials, generate connections, and manage schemas.
 
 :::note
 
-To learn how to set up your own issuer environment by deploying an issuer node, visit the [<ins>Issuer section in the documentation</ins>](./issuer/issuer-overview.md).
+When an Issuer releases a new credential type, they create a schema for that credential. This requires a set of JSON key pair data or files that gather all the attributes of that specific credential.
```

Comment:
> a set of JSON key pair data or files

"JSON" and "key pair data" are redundant. (I guess you mean "key/**value** pair data")

I'd leave something closer to the original:

> When an Issuer releases a new credential type, they create a schema for that credential (i.e. a set of JSON files that gather all the attributes of that specific credential).

Reply:
I wasn't convinced that files plural was correct base on what I had seen; seemed more likely it was data (plural) to create schema (singular) --> tried an approach that side steps both issues PTAL

Reply:
ah yes, ok plural returned